src/edu/stanford/dlss/was/WasapiDownloader.java

```diff
@@ -1,29 +1,70 @@
 package edu.stanford.dlss.was;
 
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WasapiDownloader {
   public static final String SETTINGS_FILE_LOCATION = "config/settings.properties";
 
   public WasapiDownloaderSettings settings;
 
+  private WasapiConnection wasapiConn;
+
 
   public WasapiDownloader(String settingsFileLocation, String[] args) throws SettingsLoadException {
     settings = new WasapiDownloaderSettings(settingsFileLocation, args);
   }
 
-  public void executeFromCmdLine() {
+  public void executeFromCmdLine() throws IOException {
     if (settings.shouldDisplayHelp()) {
       System.out.print(settings.getHelpAndSettingsMessage());
       return;
     }
 
-    //TODO: useful work
+    // System.out.println("DEBUG: about to request " + getFileListRequestUrl());
+    WasapiResponse wasapiResp = getWasapiConn().jsonQuery(getFileListRequestUrl());
+    // System.out.println(wasapiResp.toString());
+
+    // TODO:
+    // use WasapiCrawlSelector to select crawls with files
+    // download files for each selected crawl
```

Comment:
excellent point;  I'm going to consider it "tabled" until the next PR with the download code, as I think working on that will suggest which method(s) to create.